src/string/utf16.cr

```diff
@@ -0,0 +1,93 @@
+class String
+  # Returns the UTF-16 encoding of the given *string*.
+  #
+  # Invalid chars (in the range U+D800..U+DFFF) are encoded with the
+  # unicode replacement char value `0xfffd`.
+  #
+  # ```
+  # UTF16.encode("hi 𐂥") # => Slice[104_u16, 105_u16, 32_u16, 55296_u16, 56485_u16]
```

Comment:
`UTF16.encode("hi 𐂥")` => `"hi 𐂥".to_utf16`

Reply:
thanks, I forgot to update the docs